LayoutTests/imported/w3c/web-platform-tests/accname/name/comp_name_from_content.html

```diff
@@ -129,6 +137,14 @@ <h3 data-expectedlabel="alt-before label alt-after" data-testname="heading name
 <a href="#" data-expectedlabel="alt-before label alt-after" data-testname="link name from fallback content with ::before and ::after" class="ex fallback-before fallback-after">label</a><br>
 <br>
 
+<h1><a href="https://drafts.csswg.org/css-content/#alt">Complex Alternative Text for  CSS content (previously `alt:`)</a> in pseudo-elements</h1>
+<p>rendered text should be "before label after"</p>
+<p>accessibility label should be "start alt-before end label start alt-after end"</p>
+<button data-alt-text-before="alt-before" data-alt-text-after="alt-after" data-expectedlabel="start alt-before end label start alt-after end" data-testname="button name from complex fallback content with ::before and ::after" class="ex fallback-before-complex fallback-after-complex">label</button><br>
```

Comment:
We may add some other “complex”-ities to the tests in the future, so please make the test name include `attr()` to describe what makes it complex. It will help avoid name conflict in future test additions.